src/main/java/com/alkemy/ong/data/gateways/DefaultNewsGateway.java

```diff
@@ -28,11 +27,18 @@ public News create(News news) {
         return toModel(newsRepository.save(toEntity(news)));
     }
 
-//    @Override
-//    public List<News> findAll() {
-//        List<NewsEntity> news = newsRepository.findAll();
-//        return toModelList(news);
-//    }
+    @Override
+    public News findById(Long id) {
+        NewsEntity news = newsRepository.findById(id).orElseThrow(() -> new BadRequestException("mensaje"));
```

Comment:
podemos throwear ResourceNotFoundException en lugar de BadRequest? Asi ya mapea al resultado 404 Resource Not Found la respuesta. 
Pone un mensaje acorde en ingles!